recipes/AffjaxPostNode/README.md

```diff
@@ -0,0 +1,21 @@
+# AffjaxPostNode
+
+Performs a simple HTTP Post request using Affjax library.
+
+The post data that is sent with the request makes use of argonaut to reduce boilerplate
```

Comment:
Two comments here. Could you provide a link to `argonaut`? Otherwise, the user might not know that that's the name of a library.

Second, I'm wondering whether we should clarify that `reducing boilerplate` via `argonaut` comes with a tradeoff that Gary describes in his blog post here: https://code.slipthrough.net/2018/03/13/thoughts-on-typeclass-codecs/